packages/components/src/index.js

```diff
@@ -114,3 +115,4 @@ export {
 export { default as withNotices } from './higher-order/with-notices';
 export { default as withSpokenMessages } from './higher-order/with-spoken-messages';
 export * from './text';
+export { useTheme, default as ThemeProvider } from './theme-provider';
```

Comment:
This probably needs more time to mature as it's the first CSS in JS public API, can we start with experimental APIs?